raster-foundry/raster-foundry
# Restore color correction

## Overview

This PR restores color correction to analyses and projects.

### Checklist


## Testing Instructions

 * put a cog in a project
 * set it to single band
 * try some single band options
 * load your cog into an analysis
 * view the map
 * take down backsplash
 * set the project to a multiband color mode
 * bring up backsplash
 * make one of the tms requests with data again
 * it should all work
